project/PekkoDependency.scala

```diff
@@ -53,11 +53,11 @@ object PekkoDependency {
   }
 
   // Default version updated only when needed, https://pekko.apache.org/docs/pekko/current/project/downstream-upgrade-strategy.html
-  val minimumExpectedPekkoVersion = "1.0.0"
+  val minimumExpectedPekkoVersion = "1.0.1"
   val default = pekkoDependency(defaultVersion = minimumExpectedPekkoVersion)
   def docs = default
 
-  lazy val mainSnapshot = Artifact(determineLatestSnapshot("0.0.0"), true)
```

Comment:
This was changed to `determineLatestSnapshot("0.0.0")` because for a long period of time Pekko core had no release which caused `determineLatestSnapshot()` to fail.

Reply:
could you rebase?  #270 has been merged

Reply:
AhI see, let me rebase and also change commit

Reply:
@pjfanning Done, I have also updated the commit message to properly reflect whats going on.